packages/backend/src/payment-method/ilp/stream-credentials/service.ts

```diff
@@ -33,9 +33,6 @@ function getStreamCredentials(
   deps: ServiceDependencies,
   payment: IncomingPayment
 ): IlpStreamCredentials | undefined {
-  if (payment.isExpiredOrComplete()) {
```

Comment:
`IncomingPayment.isExpiredOrComplete()` isn't used anywhere else in the code with this deletion - perhaps it can be cleaned up?

Reply:
Given this, we should update the `receiverService.getLocalIncomingPayment` function to not error if we could not get stream credentials for the payment.